ubuntu-headless-qt/README.md

```diff
@@ -0,0 +1,5 @@
+# Script to build ubuntu 22.04 LXDE
+A core-image-qt-rzpi.tar.bz2 file needed for build script
+Put that file inside ubuntu-headless-qt folder
+
+Please remove rootfs and rootfs_qt before running main_script.sh
```

Comment:
I'm not sure what are rootfs and rootfs_qt?
As my understanding, there are two rootfs. One is the target rootfs and other is original qt rootfs. Is it correct?
If so, please change the folder name to make more sense.

Reply:
In this current script:
rootfs (target)
rootfs_qt (source)

I will change to:
rootfs (target)
qt_rootfs_source (source)
to be easier to make more sense

Reply:
And I will also remove 
~~Please remove rootfs and rootfs_qt before running main_script.sh~~
Because there is a function that handle this feature